docs/bundles/11-contributing.md

```diff
@@ -0,0 +1,22 @@
+# Contribting to Massdriver Bundles
+
+## A Note on Bundle Philosophy
+Bundles are intended to be tightly use-case scoped, intention-based reusable pieces of IaC
+for use in the Massdriver platform, see [Building the Right Size Bundle](https://docs.massdriver.cloud/bundles/development#building-the-right-sized-bundle).  
+For this reason, major feature additions that broaden the scope of and existing bundle are likely to be rejected by the community.
+
+
+There are several ways to contribute Massdriver bundles:
+- Fix bugs / minor improvements to existing bundles in the massdriver-cloud org
+- Fork an existing bundle if you are trying to make a major feature change or support a different use case with similar technology.
+- [Develop](https://docs.massdriver.cloud/bundles/development) your own bundle from scratch (optionally open source in you own org)
```

Comment:
Same with this.  I think we should have this content in the doc (about forking and/or building your own bundle), but I think it should be an "alternative" to contributing instead of under "How to contribute".

My concern is people might get the impression their forks or custom bundles will be public. That will eventually happen, but there will be a lot of vetting before we allow any bundles to be publicly usable.